knowledge-base/html5-viewer-export-operations-fail-in-chrome-and-firefox-with-err_connection_reset.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Export operations fail in Chrome and Firefox with ERR_CONNECTION_RESET
+description: Export operations fail in Chrome and Firefox with ERR_CONNECTION_RESET.
+type: how-to
+page_title: Export operations fail in Chrome and Firefox with ERR_CONNECTION_RESET
+slug: html5-viewer-export-operations-fail-in-chrome-and-firefox-with-err_connection_reset
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting HTML5 Report Viewer</td>
+		</tr>
+	</tbody>
+</table>
```

Comment:
Is there a product or browser versions associated with this error? If yes, type it here. If not, make sure to add all browsers affected.